### PR TITLE
Add image reference

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: manifests/image-references [hunk 1]
```diff
@@ -0,0 +1,8 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: cluster-bootstrap
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/origin-cluster-kube-bootstrap:v4.0
```
Comment (sttts) [inline manifests/image-references]: typo

Comment (mfojtik) [reply manifests/image-references]: was just writing that

**Pull request merged**